### PR TITLE
[jsk_fetch_startup] Use ROS_HOSTNAME and ROS_MASTER_URI

### PR DESCRIPTION
**Pull request by iory**
# What is this?

Use ROS_HOSTNAME and ROS_MASTER_URI=http://localhost:11311 instead of `rossetip`.
`rossetip` causes stopping supervisor's job.
Related to https://github.com/jsk-ros-pkg/jsk_robot/issues/1489
Reference http://wiki.ros.org/ROS/EnvironmentVariables#ROS_IP.2FROS_HOSTNAME
`Use ROS_IP if you are specifying an IP address, and ROS_HOSTNAME if you are specifying a host name.`
In the case of fetch, the hostname does not change, so I think it is better to use `ROS_HOSTNAME`.

cc: @708yamaguchi

### CODE REVIEW

**Comment by 708yamaguchi** [comment]
When `Network Manager` is restarted by `networ_monitor.py`, the `wlan*` interface become not visible.
In this situation, if we use 133.xx address, some nodes died.

```
[head_camera/depth_registered/throttle_image-58] process has finished cleanly
log file: /home/fetch/.ros/log/610d6974-e2a6-11ec-863c-67076a1ceb38/head_camera-depth_registered-throttle_image-58*.log
[head_camera/depth_registered/throttle_image-58] restarting process
process[head_camera/depth_registered/throttle_image-58]: started with pid [17554]
[head_camera/depth_registered/resize_cloud-54] process has finished cleanly
log file: /home/fetch/.ros/log/610d6974-e2a6-11ec-863c-67076a1ceb38/head_camera-depth_registered-resize_cloud-54*.log
[head_camera/depth_registered/resize_cloud-54] restarting process
process[head_camera/depth_registered/resize_cloud-54]: started with pid [17561]
[head_camera/depth_registered/relay_image-56] process has finished cleanly
log file: /home/fetch/.ros/log/610d6974-e2a6-11ec-863c-67076a1ceb38/head_camera-depth_registered-relay_image-56*.log
[head_camera/depth_registered/throttle_image_compressed-59] process has finished cleanly
log file: /home/fetch/.ros/log/610d6974-e2a6-11ec-863c-67076a1ceb38/head_camera-depth_registered-throttle_image_compressed-59*.log
[head_camera/depth_registered/throttle_camera_info-60] process has finished cleanly
log file: /home/fetch/.ros/log/610d6974-e2a6-11ec-863c-67076a1ceb38/head_camera-depth_registered-throttle_camera_info-60*.log
[head_camera/depth_registered/relay_image-56] restarting process
process[head_camera/depth_registered/relay_image-56]: started with pid [17598]
[head_camera/depth_registered/throttle_image_compressed-59] restarting process
process[head_camera/depth_registered/throttle_image_compressed-59]: started with pid [17599]
[head_camera/depth_registered/throttle_camera_info-60] restarting process
process[head_camera/depth_registered/throttle_camera_info-60]: started with pid [17600]
```

**Comment by knorth55** [comment]
@sktometometo can you check this PR?

File: jsk_fetch_robot/jsk_fetch_startup/config/config.bash [hunk 1]
```diff
@@ -20,7 +20,7 @@ if [ $(hostname) = 'fetch15' ]; then
   export RS_SERIAL_NO_L515_HEAD="";
 
   export NETWORK_DEFAULT_WIFI_INTERFACE="wlan0";
-  export NETWORK_DEFAULT_ROS_INTERFACE="133.11.216.217";
+  export NETWORK_DEFAULT_ROS_INTERFACE="localhost";
```
Comment (sktometometo) [inline jsk_fetch_robot/jsk_fetch_startup/config/config.bash:23]: `NETWORK_DEFAULT_ROS_INTERFACE` can be removed.

Comment (sktometometo) [reply jsk_fetch_robot/jsk_fetch_startup/config/config.bash:23]: If do so, please also remove [here](https://github.com/knorth55/jsk_robot/blob/58a859cfd1f55a659acde7e381b14b4416325f9f/jsk_fetch_robot/jsk_fetch_startup/README.md?plain=1#L88-L90).

Comment (knorth55) [reply jsk_fetch_robot/jsk_fetch_startup/config/config.bash:23]: I think we can leave it.

Comment (knorth55) [reply jsk_fetch_robot/jsk_fetch_startup/config/config.bash:23]: uh, no, we should remove it.

**Comment by 708yamaguchi** [comment]
Use this PR for todays demo.

> NETWORK_DEFAULT_ROS_INTERFACE can be removed.

Update this diff in the next PR.

**Comment by sktometometo** [comment]
fetch1075のAPからだと `fetch1075` では名前解決できないので、`NETWORK_DEFAULT_ROS_INTERFACE` を復活します.

**Pull request merged**